[SRVCOM-1952] Add docs on integration of Serverless with Pipelines (#68520)
openshift/openshift-docs: integrations/serverless-pipelines-integration.adoc

:_content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="serverless-pipelines-integration"]
= Integrating {ServerlessProductShortName} with {pipelines-shortname}
:context: serverless-pipelines-integration

toc::[]

> Integrating {ServerlessProductShortName} with {pipelines-shortname} enables CI/CD pipeline management for {ServerlessProductShortName} services. This allows you to automate the deployment of your {ServerlessProductShortName} services.

Reviewer: Per RH Supplementary Style guide:
> - Avoid feature-focused language. Focus on what users can accomplish with a product, not on what a product does. For example, avoid phrases such as "This product allows you to…​".
> - Use customer-centric language, such as "You can…​ by…​" or "To…​, configure…​".
```suggestion
Integrating {ServerlessProductShortName} with {pipelines-shortname} enables CI/CD pipeline management for {ServerlessProductShortName} services. Using this integration, you can automate the deployment of your {ServerlessProductShortName} services.
```
